README.md

```diff
@@ -116,6 +116,18 @@ Useful articles and learning resources.
 - [DEV.to](https://dev.to) - The All in One Blogs Portal for Developers
 </details>
 
+<details>
+  <summary>
+    🐱‍💻 Courses
+  </summary>
+
+<br />
+
+- [Github Education Pack](https://education.github.com/pack/) - Resources for programming and other
```

Comment:
Would GitHub Education Pack be under courses? It'd be more of a resource than a course, it could be under tools or another section for resources such as this